package/butterfly_angular/LICENSE

```diff
@@ -0,0 +1,24 @@
+Copyright (c) 2017, jonahwilliams.
```

Comment:
Let's keep using Apache 2.0, until/if we decide to switch.

Reply:
Changed to same Lisc info as butterfly